### ISSUE TITLE
`So...

### ISSUE THREAD

**Issue by ghost** [JFischer00/PaintWarPlugin#1]
So it's like Splatoon?

**Comment by JFischer00** [comment]
Yeah, kinda. I will warn you in case you didn't already see, I haven't worked on this project in a long time and really have no interest in it anymore. I haven't even played Minecraft in months. You're more than welcome to work on it yourself if you would like.
